client/src/views/paycheckCard/PaycheckCard.jsx

```diff
@@ -0,0 +1,137 @@
+import React, { useEffect, useState } from "react";
+import { connect } from "react-redux";
```

Comment:
If you want to level up, look into "react-redux" hooks. They are awesome :D 
https://react-redux.js.org/api/hooks